### DISCUSSION TITLE
need help

### DISCUSSION THREAD

**Discussion by satanic42** [isid0re/SoloLeveling#267] [General]
tested necro and paladin evry thing is fine in the  beginning but after few levels necro pick teeth and he stopt attacking same with paladin he do not use blessed hammer he just standing beside the monsters and letting the merc doing the job this only happen in some of the newest versions older work fine

**Comment by isid0re** [comment]
You need to make sure you updated all the files. That sounds like a bad install.

**Reply by isid0re** [reply]
Only other possibility is you dont see the attacks cuz of ForcePACKETCASTING is set to true in the developer.js file

**Reply by satanic42** [reply]
how do i update the files

**Reply by satanic42** [reply]
never mind i set ForcePACKETCASTING to false now it works thanks for the help
